extras/data/COMPASS/metadata.yaml

```diff
@@ -0,0 +1,5 @@
+COMPASS:
+  description: Measurements with Polarised beams
+  nnpdf_id: COMPASS
+  y_label: A1
```

Comment:
```suggestion
  y_label: g1
```

Reply:
it is $g_1$, right? and not $2x g_1$ (in any case it is surely not $A_1$ since this file is a grid and not a compound)